scripts/lang_sync

```diff
@@ -366,11 +366,7 @@ def post_form_upload(url, form_data, form_fentry, form_fname, fdata):
 
     # Perform the request, and return the read data.
     with urllib.request.urlopen(req) as resp:
-        if resp.status != 200:
-            return False
-
-        data = resp.read()
```

Comment:
not doing this `read()` can have side-effects. You sure this is okay? (mind you, `eints` uses `urllib`, not `urllib3` ...)

Reply:
Can it? We clearly don't care about the content of the response, and have already used the status...

Reply:
I honestly don't know, that is why I ask :D (it was an honest question). I remember a lot of quirks with `urllib`. Especially when pooling is used. So basically what I tried to ask: did you test if this is still working correctly? :D

Reply:
On line 320 something similar happens. Possibly use what they did there?